web/views/products.hbs

```diff
@@ -0,0 +1,42 @@
+<main class="flex-row">
+  <div class="category-list">
+    {{#each categories}}
+    <a href="/products/{{id}}">

```

Comment:
Is this a page for category listing? why its not named like it. I assumed it PDP page